tests/limiter/limiter.js

```diff
@@ -302,6 +303,67 @@ describe('tests/limiter/limiter.js >', () => {
     });
   });
 
+  describe('logging on 429 response >', () => {
+    // used to make done available in testLogMessage
+    let doneCopy;
+    before(() => {
+      tu.toggleOverride('enableLimiterActivityLogs', true);
+      logger.on('logging', testLogMessage);
+    });
+
+    after(() => {
+      logger.removeListener('logging', testLogMessage);
+      tu.toggleOverride('enableLimiterActivityLogs', false);
+    });
+    
+    afterEach((done) => setTimeout(done, 100));
+
+    function testLogMessage (transport, level, msg, meta) {
+      const logObj = {};
+      logObj['activity'] = msg.split(' ')[0].split('=')[1] //gets activity param from log
+      try {
+        expect(logObj.activity).to.equal('limiter');
+        doneCopy();
+      } catch (err) {
+        doneCopy(err);
+      }
+    }
+
+    it('2 quick requests', (done) => {
+      doneCopy = done;
+      makeRequest('/v1/aspects', 'post', token1)
+      .then(() => makeRequest('/v1/aspects', 'post', token1)
+      .then((res) => {
+        expect(res.status).to.equal(constants.httpStatus.TOO_MANY_REQUESTS);
+        expect(res.header['x-ratelimit-limit']).to.equal('1');
+        expect(res.header['x-ratelimit-remaining']).to.equal('0');
+      }))
+      .catch(done);
+    });
+
+    it('setup', (done) => {
+      makeRequest('/v1/aspects', 'post', token1)
+      .then((res) => {
+        expect(res.status).to.equal(constants.httpStatus.CREATED);
+        expect(res.header['x-ratelimit-limit']).to.equal('3');
+        expect(res.header['x-ratelimit-remaining']).to.equal('0');
+        done();
+      })
+      .catch(done);
+    });
+
+    it('limit over longer period', (done) => {
+      doneCopy = done
+      makeRequest('/v1/aspects', 'post', token1)
+      .then((res) => {
+        expect(res.status).to.equal(constants.httpStatus.TOO_MANY_REQUESTS);
+        expect(res.header['x-ratelimit-limit']).to.equal('3');
+        expect(res.header['x-ratelimit-remaining']).to.equal('0');
+      })
+      .catch(done);
+    });
+  })
```

Comment:
missing ;

Reply:
strange. ran it before and it didn't catch either of these. (ran it again and still doesn't complain)

Reply:
lol please update the list of directories to enforce on in the "jscs-enforce" script in package.json to include this directory